google-cloud-storage/src/main/java/com/google/cloud/storage/Crc32cUtility.java

```diff
@@ -0,0 +1,298 @@
+        0x2f8b6829,
+        0x82f63b78,
+        0x709db87b,
+        0x63cd4b8f,
+        0x91a6c88c,
+        0x456cac67,
+        0xb7072f64,
+        0xa457dc90,
+        0x563c5f93,
+        0x082f63b7,
+        0xfa44e0b4,
+        0xe9141340,
+        0x1b7f9043,
+        0xcfb5f4a8,
+        0x3dde77ab,
+        0x2e8e845f,
+        0xdce5075c,
+        0x92a8fc17,
+        0x60c37f14,
+        0x73938ce0,
+        0x81f80fe3,
+        0x55326b08,
+        0xa759e80b,
+        0xb4091bff,
+        0x466298fc,
+        0x1871a4d8,
+        0xea1a27db,
+        0xf94ad42f,
+        0x0b21572c,
+        0xdfeb33c7,
+        0x2d80b0c4,
+        0x3ed04330,
+        0xccbbc033,
+        0xa24bb5a6,
+        0x502036a5,
+        0x4370c551,
+        0xb11b4652,
+        0x65d122b9,
+        0x97baa1ba,
+        0x84ea524e,
+        0x7681d14d,
+        0x2892ed69,
+        0xdaf96e6a,
+        0xc9a99d9e,
+        0x3bc21e9d,
+        0xef087a76,
+        0x1d63f975,
+        0x0e330a81,
+        0xfc588982,
+        0xb21572c9,
+        0x407ef1ca,
+        0x532e023e,
+        0xa145813d,
+        0x758fe5d6,
+        0x87e466d5,
+        0x94b49521,
+        0x66df1622,
+        0x38cc2a06,
+        0xcaa7a905,
+        0xd9f75af1,
+        0x2b9cd9f2,
+        0xff56bd19,
+        0x0d3d3e1a,
+        0x1e6dcdee,
+        0xec064eed,
+        0xc38d26c4,
+        0x31e6a5c7,
+        0x22b65633,
+        0xd0ddd530,
+        0x0417b1db,
+        0xf67c32d8,
+        0xe52cc12c,
+        0x1747422f,
+        0x49547e0b,
+        0xbb3ffd08,
+        0xa86f0efc,
+        0x5a048dff,
+        0x8ecee914,
+        0x7ca56a17,
+        0x6ff599e3,
+        0x9d9e1ae0,
+        0xd3d3e1ab,
+        0x21b862a8,
+        0x32e8915c,
+        0xc083125f,
+        0x144976b4,
+        0xe622f5b7,
+        0xf5720643,
+        0x07198540,
+        0x590ab964,
+        0xab613a67,
+        0xb831c993,
+        0x4a5a4a90,
+        0x9e902e7b,
+        0x6cfbad78,
+        0x7fab5e8c,
+        0x8dc0dd8f,
+        0xe330a81a,
+        0x115b2b19,
+        0x020bd8ed,
+        0xf0605bee,
+        0x24aa3f05,
+        0xd6c1bc06,
+        0xc5914ff2,
+        0x37faccf1,
+        0x69e9f0d5,
+        0x9b8273d6,
+        0x88d28022,
+        0x7ab90321,
+        0xae7367ca,
+        0x5c18e4c9,
+        0x4f48173d,
+        0xbd23943e,
+        0xf36e6f75,
+        0x0105ec76,
+        0x12551f82,
+        0xe03e9c81,
+        0x34f4f86a,
+        0xc69f7b69,
+        0xd5cf889d,
+        0x27a40b9e,
+        0x79b737ba,
+        0x8bdcb4b9,
+        0x988c474d,
+        0x6ae7c44e,
+        0xbe2da0a5,
+        0x4c4623a6,
+        0x5f16d052,
+        0xad7d5351
+      };
+
+  /*
+     There is a more efficient implementation in https://github.com/google/crc32c, however for brevity we are following
+     the byte by byte computation, implemented in: https://github.com/google/crc32c/blob/main/src/crc32c_portable.cc#L252-L257
+  */
```

Comment:
Lets replace this with an actual javadoc comment for the method.

```
  /**
   * Straight forward implementation to concatenate ...
   * @param crc1
   * @param crc2
   * @param crc2ObjectSize
   * @return
   * @see <a target="_blank" rel="noopener noreferrer" href="https://github.com/google/crc32c/blob/main/src/crc32c_portable.cc#L252-L257">https://github.com/google/crc32c/blob/main/src/crc32c_portable.cc#L252-L257</a>
   */
```

We can also link to the actual code from the javadoc comment and it will be turned into a clickable link rather than simply text.